examples/hotrod/cmd/root.go

```diff
@@ -73,6 +75,9 @@ func init() {
 	RootCmd.PersistentFlags().IntVarP(&frontendPort, "frontend-service-port", "f", 8080, "Port for frontend service")
 	RootCmd.PersistentFlags().IntVarP(&routePort, "route-service-port", "r", 8083, "Port for routing service")
 
+	// Flag for serving frontend at custom basepath url
+	RootCmd.PersistentFlags().StringVarP(&basepath, "basepath", "b", "", "Basepath for frontend service")
```

Comment:
Could you please add a default value `/` it makes the help command more self explanatory.

```
  -b, --basepath string                  Basepath for frontend service (default "/")
```

instead of 
```
  -b, --basepath string                  Basepath for frontend service
```